lib/tests/streamlit/runtime/legacy_caching/hashing_test.py

```diff
@@ -888,10 +889,10 @@ def test_external_module(self):
         # will need to be updated!
 
         def call_altair_concat():
-            return altair.vegalite.v4.api.concat()
+            return altair.vegalite.api.concat()
```

Comment:
Very minor comment but in case you want to fully decouple from the lower-level submodules as per our discussion you could also import `concat` and `layer` from the top-level module:

```suggestion
            return altair.concat()
```